spec/features/users/admin/admin_merchant_can_disable_spec.rb

```diff
@@ -0,0 +1,39 @@
+require  'rails_helper'
+
+describe 'As an admin merchant' do
+  context 'when I visit the merchant index page' do
+    it "should disable an enabled merchant" do
+      @admin = create(:user, role: 2)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@admin)
+
+      @merchant_1 = create(:user, role: 1)
+      @merchant_2 = create(:user, role: 1, enabled: false)
+
+      visit admin_merchants_path
+      within "#merchant-#{@merchant_2.id}" do
+        expect(page).to have_button("enable")
+      end
+      within "#merchant-#{@merchant_1.id}" do
+        expect(page).to have_button("disable")
+        click_button "disable"
+      end
+
```

Comment:
I would check that the merchant's `enabled` value was changed at the model level here. This may already be covered in some model spec for the `switch_enabled` method but I would still add here to be thorough

Like `expect(@merchant_1.enabled).to be_truthy or eq to true`

Reply:
Not sure why the additional test fails when localhost & save_and_open_page show this as functional.